compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/types/impl/IrTypeBase.kt

```diff
@@ -19,6 +19,12 @@ import org.jetbrains.kotlin.types.model.CapturedTypeMarker
 
 abstract class IrTypeBase(val kotlinType: KotlinType?) : IrType(), IrTypeProjection {
     override val type: IrType get() = this
+
+    override fun toString(): String {
```

Comment:
This `toString` might be confusing for other implementations of IR types. Please declare it only where necessary, i.e. I guess in `IrErrorTypeImpl`.

Also, there's no need to be so verbose, you can write:
```
    override fun toString(): String = "IrErrorType($kotlinType)"
```

Reply:
This `toString` required due to the fact that we are using this type to get some data. The errors may happen event when the type will be some other inheritor of `IrTypeBase`

Reply:
Rewrite `toString` as single-line

Reply:
This toString has been aimed to show which IrType has been used where it was used (you may see in the PR where I put it into string template). So, what should I do to fix your comment issue?

Reply:
So, will remove `toString`